director/advertise.go

```diff
@@ -94,6 +94,10 @@ func parseServerAd(server Server, serverType ServerType) ServerAd {
 	}
 	serverAd.URL = *serverUrl
 
+	// We assume that the WebURL we get from topology is the same as the origin XRootD URL for now
+	// Would like to have a separate field for WebURL in the future
+	serverAd.WebURL = *serverUrl
```

Comment:
On other caveat is that right now I assume that when we fetch from OSDF Topology to populate namespace cache, we can get the WebUrl from `Server.Endpoint`, the same as XRootD URL. It worth checking if this is true or not.

Reply:
I don't think this is true.  I would simply just leave it unset (and we should ensure that, in a unit test, it's OK for this to be omitted).

1.  If the origin is not running Pelican, then there is no `WebURL`.
2. If the origin is running Pelican, then it should also advertise its own `WebURL`.